modules/install-guides/pages/install-guide-k8s.adoc

```diff
@@ -0,0 +1,324 @@
+= Installing zot with Kubernetes and Helm
+:doctype: book
+:icons: font
+:source-highlighter: highlight.js
+:sectlinks:
+
+Revised: 2022-09-01
+
+Using Kubernetes with Helm charts for zot, you can easily deploy zot as an
+application in a Kubernetes cluster.
+
+
+== Before you begin
+
+=== Prerequisites
+
+- kubectl must be installed and a Kubernetes cluster must be ready.
+- https://helm.sh/[Helm] must be installed.
+
+=== Supported platforms
+
+You can install zot on standard Linux platforms with
+Intel or ARM processors and with systemd installed.
+
+[supported-platforms-zot]
+.Supported platforms and architectures
+[%autowidth]
+|===
+| OS | ARCH | Platform
+
+| linux | amd64 | Intel-based Linux servers
+| linux | arm64 | ARM-based servers and Raspberry Pi4
+|===
+
+=== Full and minimal binary images
+
+In addition to variations for specific platforms and architectures, binary images
+are also available in full and minimal implementations:
+
+- A full zot binary image is compiled with all extensions. Extensions
+include functions such as metrics, registry synchronization, search, and scrub.
```

Comment:
This part we should cross-reference to the KB article.
Maybe say ... "To learn more about this topic, see <this>" etc